### ISSUE TITLE
"SyntaxError: Disallowed character & found in attribute value" for Vimeo embed

### ISSUE THREAD

**Issue by andrewhl** [codeschool/htmlTagValidator#20]
Hello. I'm trying to validate a Vimeo embed tag:

```html
<iframe src="https://player.vimeo.com/video/123445777?title=0&byline=0&portrait=0&badge=0" width="450" height="253" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
<p>...</p>
```

It uses & liberally. I'd like to be able to make a validation exception for this. I tried doing:

```javascript
            attributes: {
                'src': {
                    normal: /(&)/
                }
            }
```

and a couple other variations on that, without success. How can I make an exception for this?

**Comment by nwronski** [comment]
This was actually a bug and should be fixed with 30bccdc626bfac1457da990969c945fca93866f6 and available as `v1.6.0` on npm. Thanks!

**Comment by andrewhl** [comment]
@nwronski Fantastic! Thanks for the very prompt response. I'll give you a star :)
👍
